scripts/save.sh

```diff
@@ -50,6 +50,8 @@ pane_format() {
 	format+="#{pane_pid}"
 	format+="${delimiter}"
 	format+="#{history_size}"
+	format+="${delimiter}"
+	format+="#{pane_title}"
```

Comment:
I'd prefer if we grouped all `pane_` interpolations together? Maybe add `pane_title` after `pane_index`?

This will be a breaking change when performing a restore with the old resurrect file, but that's a separate concern.